src/service/checkout.ts

```diff
@@ -49,13 +50,22 @@ class Checkout extends ApiKeyAuthenticatedService {
         this._paymentsResult = new PaymentsResult(this);
     }
 
+    private setApplicationInfo(request: PaymentRequest): PaymentRequest {
+        const hasApplicationInfo = "applicationInfo" in request;
+        if(!hasApplicationInfo) {
```

Comment:
if a merchant decide to set his own applicationInfo object(what can happen if we ask them to include for example integrator name) we should always override the library version + name with the ones we provide otherwise we will loose this information. Can you adjust that this will be the case ?